src/protocols/der/all.mk

```diff
@@ -0,0 +1,15 @@
+#
+# Makefile
+#
+# Version:      $Id$
+#
+TARGET		:= libfreeradius-der$(L)
+
+SOURCES		:= base.c \
+		   decode.c \
```

Comment:
maybe include push this commit until after the `encode.c` and `decode.c` files have been added.  otherwise the build is broken.